src/components/theme-buttons.tsx

```diff
@@ -22,13 +22,17 @@ export const ThemeButtons: React.FC = () => {
   const handleInteractionsTheme = () => {
     setThemeColors("#414546", "#2f70b0");
   };
+  const handleImageTheme = () => {
+    setBackgroundImage("https://upload.wikimedia.org/wikipedia/commons/thumb/0/0c/Hubble%27s_Wide_View_of_%27Mystic_Mountain%27_in_Infrared.jpg/678px-Hubble%27s_Wide_View_of_%27Mystic_Mountain%27_in_Infrared.jpg");
+  };
```

Comment:
Not a big deal since this is just for testing, but technically the themes just refer to the primary and secondary colors that will be set by authors.  So the image background isn't a theme and this function might be clearer if it's named something like `setBackground` or `handleSetBackground` to be more consistent with the other button handlers.

Reply:
changed the function name to handleImageBackground